### PR TITLE
New links for PH

### PR DESCRIPTION
**Pull request by jakeekaj**
@casey-collab  Please review

### CODE REVIEW

File: Philippines/bago/bago.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Dito nag-uumpisa ang iyong daan tungo sa pagbabago.
+
+Bumalik sa Udacity para mag-aral o kaya bumalik sa [umpisa] (Philippines/salamat.md)
```
Comment (casey-collab) [inline Philippines/bago/bago.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Philippines/salamat.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Maligayang pasko at manigong bagong taon sa inyo! Salamat sa Udacity para sa
 
-online class na ito :D
\ No newline at end of file
+online class na ito :D
+
+Ipagpatuloy ang adventure [dito] (Philippines/bago/bago.md).
```
Comment (casey-collab) [inline Philippines/salamat.md]: This link is also invalid.

File: Philippines/bago/bago.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Dito nag-uumpisa ang iyong daan tungo sa pagbabago.
```
Comment (casey-collab) [inline Philippines/bago/bago.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jakeekaj** [comment]
@casey-collab

File: Philippines/bago/bago.md (hunk 1 above)
Comment (casey-collab) [inline Philippines/bago/bago.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Philippines/salamat.md (hunk 2 above)
Comment (casey-collab) [inline Philippines/salamat.md]: This link is also invalid.

File: Philippines/bago/bago.md (hunk 3 above)
Comment (casey-collab) [inline Philippines/bago/bago.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**